client/my-sites/importer/site-importer/site-importer-input-pane.jsx

```diff
@@ -419,7 +426,7 @@ class SiteImporterInputPane extends React.Component {
 						<SiteImporterSitePreview
 							siteURL={ this.state.importSiteURL }
 							importData={ this.state.importData }
-							isLoading={ this.state.loading }
+							isLoading={ this.state.loading || this.state.forcePreviewLoadingState }
```

Comment:
If you don't introduce this extra flag and change this line to check the current importer state, like 
```
							isLoading={ this.state.loading || this.props.importerStatus.importerState == appStates.UPLOAD_SUCCESS }
```
It seems to solve it for me. I've only tested it on single author sites though, so it may be that this doesn't work in every case.

Reply:
Nice! I'll give this a run, cheers @pablinos :)